po/r-novice-gapminder.ja.po

```diff
@@ -600,7 +600,7 @@ msgstr ""
 "Except where otherwise noted, the example programs and other software\n"
 "provided by Software Carpentry and Data Carpentry are made available under the\n"
```

Comment:
Just spotted a passage that wasn't translated.

```suggestion
"Except where otherwise noted, the example プログラム and other ソフトウェア\n"
"provided by <ruby>Software Carpentry<rp>【</rp><rt>ソフトウェアカーペントリー</rt><rp>】</rp></ruby>と<ruby>Data Carpentry<rp>【</rp><rt>データーカーペントリー</rt><rp>】</rp></ruby>are made available under the\n"
```

Reply:
This is a work in progress (help welcome). https://github.com/swcarpentry-ja/i18n/pull/127/files#diff-627559538775bc873d27f74c80ae4d17L600-R604

Reply:
See the updated version here: https://github.com/swcarpentry-ja/i18n/pull/127/commits/194169ffdb3ff34ad4d9e4b635e67a920d6e1f45

```diff
- "Except where otherwise noted, the example programs and other software\n"
- "provided by Software Carpentry and Data Carpentry are made available under the\n"
- "[OSI][osi]-approved\n"
- "[MIT license](https://ja.osdn.net/projects/opensource/wiki/licenses%2FMIT_license)."
+ "Except where otherwise noted, the example プログラム and other ソフトウェア\n"
+ "provided by <ruby>Software Carpentry<rp>【</rp><rt>ソフトウェアカーペントリー</rt><rp>】</rp></ruby>と\n"
+ "<ruby>Data Carpentry<rp>【</rp><rt>データーカーペントリー</rt><rp>】</rp></ruby>は\n"
+ "[OSI][osi]の検定済みの\n"
+ "[MIT license](https://ja.osdn.net/projects/opensource/wiki/licenses%2FMIT_license)の下で利用可能です。"
```

Reply:
Translated with help from @masamiy ```msgstr ""
"特に記載がある場合を除いて、ソフトウェアカーペントリーおよびデータカーペントリーが提供しているサンプルプログラムやソフトウェア\n"
は、
"<ruby>Data Carpentry<rp>【</rp><rt>データーカーペントリー</rt><rp>】</rp></ruby>は\n"
"[OSI][osi]が承認した\n"
"[MIT ライセンス](https://ja.osdn.net/projects/opensource/wiki/licenses%2FMIT_license)の下で利用可能です。"```